apps/studio/components/interfaces/Auth/Policies/PolicyTableRow/index.tsx

```diff
@@ -47,6 +50,15 @@ export const PolicyTableRow = ({
     .filter((policy) => policy.schema === table.schema && policy.table === table.name)
     .sort((a, b) => a.name.localeCompare(b.name))
 
+  const { data: lints = [] } = useProjectLintsQuery({ projectRef: project?.ref })
+  const [rlsLint] = lints.filter(
+    (lint) =>
+      lint.categories.includes('SECURITY') &&
+      lint.metadata?.name === table.name &&
+      lint.metadata?.schema === table.schema &&
+      (lint.name === 'rls_enabled_no_policy' || lint.name.includes('rls_disabled'))
```

Comment:
`lint.name.includes('rls_disabled')` may not be forward-compatible. We could add a new rule with `rls_disabled` which means something else.